docs/on-premise.md

```diff
@@ -1,3 +1,6 @@
+---
+meta_description: "This section is dedicated to the Image-Charts paying On Premise version. It will give you all the functionalities of this subscription."
```

Comment:
"Deploy and host Image-Charts inside your own infrastructure with our On-Premise docker container."